js-exercises/curry/curry.js

```diff
@@ -0,0 +1,21 @@
+function curry(fn) {
+  if (typeof fn !== 'function') {
+    throw new TypeError(`Expected function, got ${typeof fn}, please try again with a function argument`);
+  }
+  return function internalCurrying(...args) {
```

Comment:
What we are returning is a Curried Function. Therefore a better name would have been `curriedFn`.

Reply:
Thanks @ahhda, I'll make these changes. Yes, curriedFn is sounding better.